### PR TITLE
Feature/simple eyetracker

### PR DESCRIPTION
**Pull request by Hnkee**

### CODE REVIEW

File: source/Eyetracking.NET/Eyetracker.cs [hunk 1]
```diff
@@ -1,18 +1,56 @@
-﻿using Eyetracking.NET.Tobii;
+﻿using System;
+using Eyetracking.NET.Tobii;
 
 namespace Eyetracking.NET
 {
-    public class Eyetracker 
+    public class Eyetracker : IEyetrackerVr, IEyetracker
```
Comment (dansav) [inline source/Eyetracking.NET/Eyetracker.cs]: If I write an application using this class, how can I know if it  is using a VR tracker or a desktop tracker? I guess it opens up for a bit of unexpected behavior.

**Review by dansav — COMMENTED** [review]
See inline comment

File: source/Eyetracking.NET/Eyetracker.cs (hunk 1 above)
Comment (Hnkee) [reply source/Eyetracking.NET/Eyetracker.cs]: Yes, it might be better to always have them separated.

**Comment by Hnkee** [comment]
Skip for now, use only factory

**Pull request closed without merging**